scamp/init.go

```diff
@@ -20,7 +20,7 @@ func Initialize(configPath string) (err error) {
 		return
 	}
 
-	defaultCache, err = newServiceCache(cachePath)
+	defaultCache, err = NewServiceCache(cachePath)
```

Comment:
Why are we re-exporting serviceCache and newServiceCache?

Reply:
Need it for searching for actions in a sector right now, since the service cache isn't exposed anywhere.

Reply:
Ok sounds good. Let's make a note to expose an exported function for that. SearchActionBySector() or something like that